libs/utils/utils.js

```diff
@@ -93,7 +103,7 @@ export const [setConfig, getConfig] = (() => {
   return [
     (conf) => {
       const { origin } = window.location;
-      config = { env: getEnv(), ...conf };
+      config = { env: getEnv(conf), ...conf };
```

Comment:
nit: since you're passing `conf` into getEnv, it may as well be merged in that function instead